migration/mule/foreach-processing-and-choice-routing/README.md

```diff
@@ -0,0 +1,53 @@
+# Foreach Processing and Choice Routing #
+
+This example explains how to perform content based routing along with processing per each item in the payload. 
+
+### Content Based Routing ###
+The Content-Based Router reads the content of a message and routes it to a specific recipient based on its content. It can evaluate request url, request payload or a part of it and route it to a specific path. 
+
+### Foreach Processing ###
+This concept can be achived using the WSO2 Foreach Mediator. It requires an XPath expression and a sequence (inline or referred). It splits the message into a number of different messages derived from the original message by finding matching elements for the XPath expression specified. Based on the matching elements, new messages are created for each iteration and processed sequentially. The processing is carried out based on a specified sequence. The behaviour of ForEach mediator is similar to a generic loop. After mediation, the sub-messages are merged back to their original parent element in the original message sequentially.

```

Comment:
`After mediation, the sub-messages are merged back to their original parent element in the original message sequentially.`
Shall we remove  `sequentially ` part here?

Reply:
Done